examples/with-styled-components-rtl/package.json

```diff
@@ -8,13 +8,13 @@
   },
   "dependencies": {
     "next": "latest",
-    "react": "16.12.0",
-    "react-dom": "16.12.0",
-    "styled-components": "5.0.0",
-    "stylis-plugin-rtl": "1.0.0"
```

Comment:
Should this have been deleted?

Reply:
Sorry, I forgot the last commit, now it's correct